lib/checks/color/color-contrast-evaluate.js

```diff
@@ -9,9 +9,26 @@ import {
 	getBackgroundColor,
 	getForegroundColor,
 	incompleteData,
-	getContrast
+	getContrast,
+	getOwnBackgroundColor
 } from '../../commons/color';
 
+function hasPsuedoElement(node, pseudo) {
```

Comment:
Would it be worth memoizing this method? Have you tested the performance of this?